readme.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/weg-li/weg-li.svg?branch=master)](https://travis-ci.org/weg-li/weg-li)
+![Build Status](https://github.com/MarlonSchultz/weg-li/workflows/Rails%20Tests%20Ubuntu/badge.svg)
```

Comment:
For the main repo that should be: 

https://github.com/weg-li/weg-li/workflows/Rails%20Tests%20Ubuntu/badge.svg

Reply:
😆 thx